operations/template/app.tf

```diff
@@ -183,9 +185,8 @@ resource "azurerm_linux_web_app_slot" "pre_live" {
   }
 
   app_settings = {
```

Comment:
Under `site_config`, we need to add...
```
application_stack {
   docker_registry_url = "https://${azurerm_container_registry.registry.login_server}"
   docker_image_name   = "ignore_because_specified_later_in_deployment"
 }
```

Reply:
We also need to add the `lifecycle` back, preferably at the bottom.
```
lifecycle {
 ignore_changes = [
   site_config[0].application_stack[0].docker_image_name,
   # Ignore changes to tags because the CDC sets these automagically
   tags,
 ]
}
```